core-bundle/contao/dca/tl_files.php

```diff
@@ -219,25 +219,25 @@
 		(
 			'inputType'               => 'text',
 			'eval'                    => array('rgxp'=>'digit', 'nospace'=>true, 'tl_class'=>'w50 clr'),
-			'sql'                     => "DOUBLE unsigned NOT NULL default 0"
+			'sql'                     => "DOUBLE unsigned NOT NULL default '0'"
```

Comment:
Would this work, too?

```suggestion
			'sql'                     => "DOUBLE unsigned NOT NULL default 0.00"
```

Reply:
I tried `0` and `0.0`, they did not work.

Reply:
But it does not make sense that `default 0` does not work and `default '0'` does. Especially since `default 0` works for `int` fields.

Reply:
Yeah this sounds like either a bug in Doctrine or a problem in our DcaSchemaProvider?

Reply:
Fixed in 3a5b7558da216494eeecc85aea47606aa0a2e118

Reply:
Unfortunately the `DcaSchemaProviderTest` fails now :/